programs/survival/auto_lighter.sc

```diff
@@ -37,6 +37,10 @@ global_effect_radius = 128;
 
 global_survival=false;
 
+global_min_light_level = if(system_info('game_major_target')>=18, 1, 8); //1 for 1.18+, 8 for 1.17 and earlier
```

Comment:
Maybe extract the logic to a function like get level for version, so if someone is looking at it it doesn't seem like they shouldn't touch it or it'll break.

Reply:
sure